mqsnoop/README.md

```diff
@@ -0,0 +1,87 @@
+# mqsnoop
+
+This traces POSIX message queue send.
+This is done by instrumenting /lib64/libc.so.6:mq_timedsend uprobe.
```

Comment:
This line needs to get updated (or just deleted)

Reply:
Deleted, thank.